MovieAppPetProject/MovieAppPetProject/Watchlist/WatchlistViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  WatchlistViewModel.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/05/14.
+//
+
+import Foundation
+
+class WatchlistViewModel {
+
+    // MARK: - Variable
+    private let coreDataManager = CoreDataManager()
```

Comment:
Lets rather inject CodeData in the viewModel

Reply:
will do